src/app/app.module.ts

```diff
@@ -30,10 +30,7 @@ import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
     BrowserAnimationsModule
   ],
   providers: [],
-  bootstrap: [AppComponent],
-  entryComponents: [
```

Comment:
@flavens We can then safely remove `entryComponents` from https://github.com/dasch-swiss/knora-ui-ng-lib/blob/714826ad197eea8dd16286510e65c39b1d1a8aec/projects/knora-ui/src/lib/viewer/viewer.module.ts#L80